data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project.
+#   Settings are meant to be controlled via OpenQA variables and managed by test:
+#   tests/sles4sap/publiccloud/qesap_ansible.pm
+# https://github.com/SUSE/qe-sap-deployment
```

Comment:
What is purpose to add the link to the qe-sap project here? Maybe a better place is line 6

Reply:
No particular reason. I can put it on line ... maybe 7?  Line 6 will become quite long if I put the link in there. 
But in general... I don't have preference. It's just the 120ch line python habit :D

Reply:
Appended to L6